app/controllers/answer.rb

```diff
@@ -0,0 +1,11 @@
+post '/answers' do
+  @answer = Answer.new(choice_id: params[:answer])
+  if request.xhr?
+    @answer[:question_id] = @answer.choice.question.id
+    @answer[:user_id] = session[:user_id]
+    @answer[:survey_id] = @answer.question.survey.id
+    @answer.save
```

Comment:
Right now it looks like you're responding to this AJAX request with a Ruby answer object. Is there something more meaningful you can respond with?